packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -656,7 +656,7 @@ describe("typespec-client-generator-core: package", () => {
 
       const fooBarAccessor = fooClient.methods[0];
       strictEqual(fooBarAccessor.kind, "clientaccessor");
-      strictEqual(fooBarAccessor.crossLanguageDefintionId, "TestService.Foo.Bar");
+      strictEqual(fooBarAccessor.crossLanguageDefintionId, "TestService.Foo.getBar");
```

Comment:
why is the cross language definition id changing here?

Reply:
they should come from some local cache problem, i rollbacked the changes.